core/src/com/cloud/agent/api/storage/CopyVolumeCommand.java

```diff
@@ -75,4 +76,12 @@ public boolean toSecondaryStorage() {
     public String getVmName() {
         return vmName;
     }
+
+    public String getNfsVersion() {
+        return nfsVersion;
+    }
+
+    public void setNfsVersion(String nfsVersion) {
+        this.nfsVersion = nfsVersion;
+    }
```

Comment:
This looks like a useful change to the storage API, but why is it necessary to implement OSPF?  Seems like it should be in a separate PR.